treemacs-branch-creation.el

```diff
@@ -178,9 +178,14 @@ correct cache entries."
           (button-put b 'abs-path (nth (- (length it) 1) it))
           (button-put b 'parent-path root)
           (end-of-line)
-          (let ((beg (point)))
-            (insert (cadr it))
-            (add-text-properties beg (point) props)))))))
+          (let* ((beg (point))
+		 (dir (cadr it))
```

Comment:
Indentation looks wrong.

Reply:
I deleted tabs. We should set indent-tabs-mode to nil in .dir-locals.el because by default it's enabled.

Reply:
I'll put adding file-local indent-tabs-mode=nil declarations on my todo list.